models.py

```diff
@@ -6,10 +6,11 @@
 from kytos.core import log
 from kytos.core.helpers import now, get_time
 from kytos.core.interface import UNI
+from kytos.core.common import GenericEntity, EntityStatus
 from napps.kytos.mef_eline import settings
 
 
-class EVC:
+class EVC(GenericEntity):
     """Class that represents a E-Line Virtual Connection."""
 
     def __init__(self, **kwargs):
```

Comment:
@macartur I believe the active attribute should be instantiated as False, can we make this explicitly in the code? It could lead to some issues if the EVC gets created as active until it's been actually deployed.